src/annotator/delegator.js

```diff
@@ -0,0 +1,95 @@
+import $ from 'jquery';
+
+// Adapted from:
+//  https://github.com/openannotation/annotator/blob/v1.2.x/src/class.coffee
+//
+//  Annotator v1.2.10
+//  https://github.com/openannotation/annotator
+//
+//  Copyright 2015, the Annotator project contributors.
+//  Dual licensed under the MIT and GPLv3 licenses.
+//  https://github.com/openannotation/annotator/blob/master/LICENSE
+
+/**
+ * `Delegator` is a base class for many objects in the annotator.
+ *
+ * It provides:
+ *
+ *  - An event bus, attached to the DOM element passed to the constructor.
```

Comment:
In practice across annotator classes, is this element always the same root element? Curious.

Reply:
All the classes which need to publish or subscribe to events will be sharing the same root element, I haven't checked if those classes which don't deal with events at all are.